### PR TITLE
Trailing commas (comma-dangle) and minor cleanups

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,16 +16,17 @@
     "url": "https://github.com/wireapp/wire-desktop.git"
   },
   "devDependencies": {
-    "electron-mocha": "2.3.1",
-    "electron-packager": "7.3.0",
+    "electron-mocha": "3.0.5",
+    "electron-packager": "8.0.0",
     "electron-prebuilt": "1.2.8",
```
Comment (herrmannplatz) [inline package.json]: have you checked if this breaks stuff, since its a major version bump?

File: package.json [hunk 2]
```diff
@@ -16,16 +16,17 @@
...
+    "eslint": "3.4.0",
+    "eslint-config-google": "^0.6.0",
     "grunt": "1.0.1",
     "grunt-contrib-clean": "1.0.0",
     "grunt-electron-installer": "2.1.0",
     "grunt-git": "1.0.0",
     "grunt-github-changes": "0.1.0",
-    "load-grunt-tasks": "3.5.0"
+    "load-grunt-tasks": "3.5.2"
   }
```
Comment (herrmannplatz) [inline package.json]: isnt 

```
,
    "eslintConfig": {
        "extends": "google"
    }
```

missing in order to apply the config?

File: electron/main.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ let baseURL = argv.env || (config.PRODUCTION ? config.PRODUCTION_URL : config.IN
 // Misc
 ///////////////////////////////////////////////////////////////////////////////
 raygunClient = new raygun.Client().init({
-  apiKey: config.RAYGUN_API_KEY
+  apiKey: config.RAYGUN_API_KEY,
 });
```
Comment (herrmannplatz) [inline electron/main.js]: should we make it a one liner as well?

File: package.json (hunk 2 above)
Comment (lipis) [inline package.json]: oups.. I was playing around with stuff.. shouldn't been added there :)

Comment (herrmannplatz) [inline package.json]: ah ok

File: package.json (hunk 1 above)
Comment (lipis) [inline package.json]: Works.. macOS and Windows.. Checked..

**Pull request merged**